### PR TITLE
feat: Load all input data from zip archives instead of unpacking the files before

### LINKED ISSUE #161: Read input from zip directly
Going through the input files, we now use almost always CSV and XLSX which are in a zip. Like we do for SIRENE, we can simply directly read from the ZIP instead of asking the user to extract the data. 

- [ ] Go through the data sets and note where we ask the user to unpack the data
- [ ] Change the code so it uses the zip directly

### LINKED ISSUE #164: Replace regional RP data with national
INSEE provides a CSV with all French regions instead of the regional ones. We should base the pipeline on the national one (500MB vs. 100MB of the individual ones) to simplify the process of passing to other territories than IDF.

### PR DESCRIPTION
**Pull request by sebhoerl**
This PR makes the data collection process easier by loading all data diretly from the archives instead of requiring the user to unpack all the ZIP files downloaded from INSEE. Also, we pass to the national census data (which is only 500MB compares to the ~100M  for the regionalized one) to ease the creation of other scenarios than IDF.

Closes #161. Closes #164.

### CODE REVIEW

File: config_toulouse.yml [hunk 1]
```diff
@@ -30,8 +30,7 @@ config:
   java_memory: 14G
 
   # Nantes settings
```
Comment (Nitnelav) [inline config_toulouse.yml:32]: # Toulouse settings ?

**Comment by Nitnelav** [comment]
Yeah I remember trying to update dependencies to python 3.7+ a few month ago and I gave up after a few days. Maybe I can help test/fix stuff on my machines if you want or if you're stuck. Good luck !

**Comment by sebhoerl** [comment]
It was a long battle, here is the current status:
- I added a Windows pipeline to Github CI, so commits are now automatically tested in a Windows envrionment as well
- There is now an `environment.yml` that is entirely based on *conda-forge* an works both on Linux and Windows, at least in CI

@aburianne @Nitnelav @tkchouaki
Could you please test if you can install this environment on your local (Windows) machines and then check whether you can do `import fiona` without any error?

**Comment by Nitnelav** [comment]
it works on my machine ! Really nice job 👍 
image

**Pull request merged**